sql-statements/sql-statement-commit.md

```diff
@@ -39,6 +39,7 @@ Query OK, 0 rows affected (0.01 sec)
 
 ## MySQL compatibility
 
+* TiDB does not currently use Metadata Locking (MDL) to prevent DDL statements from modifying tables that are in use by transactions. In the event that table a definition has changed, committing a transaction will result in an `Information schema is changed` error. In this event, the transaction is rolled back automatically.
```

Comment:
```suggestion
* Currently, TiDB does not use Metadata Locking (MDL) to prevent DDL statements from modifying tables used by transactions. If the definition of a table has changed, committing a transaction will result in an `Information schema is changed` error. In this event, the transaction is rolled back automatically.
```
What does "table a definition" mean?

Reply:
It should just be "table definition". It means the columns, names of the columns etc.